common/lazy-load-assemblies-wasm/readme.md

```diff
@@ -0,0 +1,42 @@
+# Telerik Blazor Components and WebAssembly Lazy Load of Assemblies
+
+This sample project demonstrates how you can use the <a href="https://docs.microsoft.com/en-us/aspnet/core/blazor/webassembly-lazy-load-assemblies?view=aspnetcore-5.0" target="_blank">lazy assembly loading feature of Blazor</a> with the Telerik components.
+
+## Requirements
+
+There are a few key points and changes to a standard project with relation to the Telerik components:
+
+1. Move the `<TelerikRootComponent>` to a layout that is used only on pages that have the Telerik assemblies loaded.
+
+1. Mark the required assemblies as lazy-loaded in the `.csproj` file of the web assembly project - the following snippet shows the assemblies that the Telerik components rely on at the time of the `2.19.0` release. 
```

Comment:
We can make it clear that not all of these assemblies need to be lazy loaded (referring to the part "Mark the required assemblies") - it is up to the developer to decide, but they can choose a subset of that. For example, `System.Linq.Queryable` is a "common" package that might have been already loaded.